pmd-java/src/test/resources/net/sourceforge/pmd/lang/java/rule/multithreading/xml/AvoidSynchronizedStatement.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<test-data
+    xmlns="http://pmd.sourceforge.net/rule-tests"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://pmd.sourceforge.net/rule-tests http://pmd.sourceforge.net/rule-tests_1_0_0.xsd">
+
+    <test-code>
+        <description>TEST1</description>
+        <expected-problems>1</expected-problems>
+        <code><![CDATA[
+public class Foo {
+    void foo () {
+        synchronized(mutex) {}
+    }
+}
+        ]]></code>
+    </test-code>
+
+    <test-code>
+        <description>#991 AvoidSynchronizedStatement for static methods</description>
+        <expected-problems>1</expected-problems>
+        <code><![CDATA[
+public class Test {
+    public static void foo() {
+        synchronized(Test.class) {
+            // only a block is synchronized on Test.class
+        }
+    }
+}
+        ]]></code>
+    </test-code>
```

Comment:
To make sure, we don't overlap with [AvoidSynchronizedAtMethodLevel](https://docs.pmd-code.org/latest/pmd_rules_java_multithreading.html#avoidsynchronizedatmethodlevel)

```suggestion
    </test-code>
    
    <test-code>
        <description>synchronized methods are not flagged - we have a separate rule AvoidSynchronizedAtMethodLevel for that</description>
        <expected-problems>0</expected-problems>
        <code><![CDATA[
public class Test {
    public synchronized void foo() {}
    public static synchronized void fooStatic() {}
}
        ]]></code>
    </test-code>
```